pages.es/linux/newgrp.md

```diff
@@ -0,0 +1,11 @@
+# newgrp
+
+> Cambia el grupo primario de pertenencia.
+
+- Cambia el grupo primario de pertenencia del usuario:
+
+`newgrp {{nombre_grupo}}`
+
+- Restablece el grupo primario de pertenencia al grupo del usuario predefinido en `/etc/passwd`:
```

Comment:
```suggestion
- Restablece el grupo primario de pertenencia al grupo por defecto del usuario `/etc/passwd`:
```

Reply:
What do you think about "por omisión"?

Reply:
I think `default` -> `por defecto`, it's a standard translation <https://apagayenciende.net/que-significa-por-defecto/>.